unit/java_bytecode/java_bytecode_parser/java_bytecode_convert_class_lambda_method_handles.cpp

```diff
@@ -0,0 +1,254 @@
+/*******************************************************************\
+
+ Module: Unit tests for parsing generic classes
+
+ Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+#include <algorithm>
+
+#include <testing-utils/catch.hpp>
+#include <testing-utils/load_java_class.h>
+#include <testing-utils/run_test_with_compilers.h>
+#include <testing-utils/require_type.h>
+
+SCENARIO(
```

Comment:
These cover checking that the method handle table ends up in the symbol table, it would be great to have tests that check the indexs for the instruction end up in the instruction, but maybe this is better covered by the next step when we've actually converted that instruction.

Reply:
Yes, I think it will be easier in the next step. If you would prefer to have tests at this stage, let me know.